data/data.js

```diff
@@ -1,14 +1,19 @@
+import path from 'path';
+import {fileURLToPath} from 'url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
 import inquirer from 'inquirer';
 import chalk from 'chalk';
 import tmp from "tmp";
 
 /** Load configuration */
 import fs from 'fs'
-const stringConfig = fs.readFileSync(process.cwd() + "/../gh-edu/config.json", { encoding: "utf8" })
+const stringConfig = fs.readFileSync(__dirname + "/../../gh-edu/config.json", { encoding: "utf8" })
```

Comment:
The code `/../../gh-edu/config.json` is error-prone and depends on the operating system. Must be changed to `path.join(...)`etc.

Reply:
I'm finishing the conversion from js to ts in the core. When I'm done I'll check this PR.